client/my-sites/plans-features-main/index.jsx

```diff
@@ -379,6 +388,44 @@ export class PlansFeaturesMain extends Component {
 		return false;
 	}
 
+	renderProductsSelector() {
+		if ( ! isEnabled( 'plans/jetpack-backup' ) ) {
+			return null;
+		}
+
+		const { intervalType, siteId, translate } = this.props;
+		const products = [
+			{
+				title: 'Jetpack Backup',
+				description: (
+					<p>
+						Automatic scanning and one-click fixes keep your site one step ahead of security
+						threats. <a href="https://jetpack.com/">More info</a>
+					</p>
+				),
+				id: PRODUCT_JETPACK_BACKUP,
+				options: {
+					yearly: [ PRODUCT_JETPACK_BACKUP_DAILY, PRODUCT_JETPACK_BACKUP_REALTIME ],
+					monthly: [
+						PRODUCT_JETPACK_BACKUP_DAILY_MONTHLY,
+						PRODUCT_JETPACK_BACKUP_REALTIME_MONTHLY,
+					],
+				},
+				optionsLabel: 'Backup options',
+			},
+		];
+
+		return (
+			<Fragment>
+				<FormattedHeader
+					headerText={ translate( 'Single Products' ) }
+					subHeaderText={ translate( 'Just looking for a backups? We’ve got you covered.' ) }
```

Comment:
`a backups` seems off. Should it be `a backup` or just `backups`?

Reply:
Should we be translating these string already? Have they been 100% approved?

Reply:
Yep, I asked that question in https://github.com/Automattic/wp-calypso/pull/36951#discussion_r337922220

Reply:
Not sure what's the answer here, but we should be consistent IMHO.

Reply:
As discussed via Slack, let's leave all new strings untranslated for now.